schema/pet-insurance.json

```diff
@@ -152,17 +201,649 @@
                 "petType": {
                     "type": "string",
                     "description": "",
-                    "example": "dog"
+                    "example": "dog",

```

Comment:
The name of the key is "petType", which is inside "Pets". This property is clearly part of a pet, so it doesn't need to be prefixed with "pet".
Try to find a better key name than "type".

Reply:
I'll fix.